oqsprov/oqsprov_keys.c

```diff
@@ -830,7 +830,7 @@ int oqsx_key_maxsize(OQSX_KEY *key) {
     case KEY_TYPE_SIG:
 	return key->oqsx_provider_ctx.oqsx_qs_ctx.sig->length_signature;
     case KEY_TYPE_HYB_SIG:
-	return key->oqsx_provider_ctx.oqsx_qs_ctx.sig->length_signature + key->oqsx_provider_ctx.oqsx_evp_ctx->evp_info->length_signature;
+	return key->oqsx_provider_ctx.oqsx_qs_ctx.sig->length_signature + key->oqsx_provider_ctx.oqsx_evp_ctx->evp_info->length_signature+SIZE_OF_UINT32;
```

Comment:
Does this reflect a bug in the earlier code?

Reply:
"Reflect". Nice term. Yes: The previous hybrid sig code was buggy, indeed.